docsk8s/quick-start/ls-k8s-quick-start.asciidoc

```diff
@@ -0,0 +1,202 @@
+[[ls-k8s-quick-start]]
+== Quick start
+
+WARNING: This documentation is still in development and may be changed or removed in a future release.
+
+This guide describes how to set up {ls} to deliver Kubernetes logs to {es}. The logs will be monitored by Filebeat, processed through a Logstash pipeline, and then delivered into an {es} cluster in the Kubernetes environment.
+
+This section includes the following topics:
+
+* <<qs-prerequisites>>
+* <<qs-set-up>>
+* <<qs-generate-certificate>>
+* <<qs-create-elastic-stack>>
+* <<qs-view-monitoring-data>>
+* <<qs-tidy-up>>
+* <<qs-learn-more>>
+
+[float]
+[[qs-prerequisites]]
+=== Prerequisites
+
+Before you start, there are a few things you'll need:
+
+. A running Kubernetes cluster - For single node testing we recommend using link:https://minikube.sigs.k8s.io[Minikube], which allows you to easily run a single node Kubernetes cluster on your system. Check the `Getting Started` section for install and set up instructions.
+. A link:https://github.com/elastic/logstash/blob/feature/kubernetes/k8s/recipes/logstash-k8s-quickstart.zip[small zip file] of config files - Download and expand this archive into an empty directory on your local system. The files are described in <<sample-configuration-files,Sample configuration files>>.
```

Comment:
For now, this link to the zip file points to the K8s feature branch, so we'll need to fix this before or when that branch is merged.